s4e-web/cypress/support/commands.js

```diff
@@ -23,3 +23,10 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+import * as unfetch from '../dependencies/unfetch.umd';
+
+Cypress.on('window:before:load', win => {
```

Comment:
If you're using some walk around's write why you're using it. The shorter and simpler description, the better.

Reply:
I added commend 
// allows support fetch method

Reply:
I would likely see it like this

```
// TODO: Remove after Cypress upgrade to version above 5.x.y
/**
 * UnfetchJS library support data (XML) fetching incorrectly working in the cypress 4.x.y 
 * and fixed in the latest versions of it
 */
Cypress.on('window:before:load', win => {...});

Reply:
added